rj_vision_receiver/src/vision_receiver.cpp

```diff
@@ -35,7 +27,29 @@ VisionReceiver::VisionReceiver()
       param_provider_(this, kVisionReceiverParamModule) {
     recv_buffer_.resize(65536);
 
-    set_port(PARAM_vision_interface, PARAM_port);
+    /* below, vision_interface should be IP where vision receiver pubs to (in
+     * scrim-2022 this was same IP as ext ref)
+     *
+     * run ifconfig to see list of interfaces on this computer
+     *
+     * port=10006 for real vision (per SSL rulebook)
+     * port=10020 for ER-force sim
+     *
+     * see PR #1887 for last time this file was used w/ external interface
+     */
+
+    // TODO(Kevin): since these are set on init, this does not allow us to
+    // dynamic reconfigure these values, which would be nice
+
+    // port number for vision receiver
+    this->get_parameter("port", param_port_);
+    // The hardware interface to use.
+    this->get_parameter("interface", param_vision_interface_);
+    SPDLOG_ERROR("port {}", param_port_);
+    SPDLOG_ERROR("vision_interface {}", param_vision_interface_);
```

Comment:
shouldn't be ERROR level debugs, either drop to INFO and clarify the msg or delete